next.config.js

```diff
@@ -1,8 +1,11 @@
-module.exports = {
-  future: {
-    webpack5: true,
-  },
+/**
+ * @type {import('next').NextConfig}
+ */
+ const nextConfig = {
```

Comment:
No need to assign the object to nextConfig and then exporting it, let it be the way it was before, just remove the future object